dependencies.py

```diff
@@ -0,0 +1,146 @@
+# This file is updated by manage_depencies.py
```

Comment:
This file, along with `circle_config_init.py`/`generate-circleci-config.py`/`manage_dependencies.py` should not be in the root of the repo.